### DISCUSSION TITLE
self-hosting documentation from https://docs.getoutline.com

### DISCUSSION THREAD

**Discussion by johnny-smitherson** [outline/outline#6611] [Self Hosting] [answered]
The main documentation page is login-walled. This means users in offline environments can't use it at all, and users in online environments have to provide PII to see the documentation.

Is there a way to mirror the documentation website in a self-hosted instance? I don't see any `outline-docs` repo on GitHub, or the source code anywhere. Is the documentation open at all?

**Comment by tommoor** [comment, marked as answer]
This was a mistake, the guides are restored now

**Reply by johnny-smitherson** [reply]
<img width="907" alt="aaCapture" src="https://github.com/outline/outline/assets/127537716/39145655-eef1-4fe0-8a99-19a42cd5635f">

**Comment by johnny-smitherson** [comment]
My question is about offline use (providing the docs alongside an installation for environment without internet) - if the docs site is not available to import in a self-hosted environment, maybe a PDF export exists?

**Reply by tommoor** [reply]
The documentation is hosted in our own instance of Outline, not in a repo and updated almost every day. There isn't a plan to provide an offline version – this was a one off bug that caused it to be inaccessible for the first time in 5 years 😉
